openmc/capi/tally.py

```diff
@@ -190,6 +215,30 @@ def active(self):
         _dll.openmc_tally_get_active(self._index, active)
         return active.value
 
+    @property
+    def type(self):
+        type = c_int32()
+        _dll.openmc_tally_get_type(self._index, type)
+        return _TALLY_TYPES[type.value]
+
+    @type.setter
+    def type(self, type):
+        _dll.openmc_tally_update_type(self._index, type.encode())
+
+    @property
+    def estimator(self):
+        estimator = c_int32()
+        try:
+            _dll.openmc_tally_get_estimator(self._index, estimator)
+        except AllocationError:
+            return ""
```

Comment:
I would actually just let the exception be raised in this case (i.e., no need for try/except).